src/compiler/diagnosticMessages.json

```diff
@@ -3897,10 +3897,6 @@
         "category": "Message",
         "code": 95002
     },
-    "Extract symbol": {

```

Comment:
I know some teams have a convention of leaving a comment behind so that the code doesn't appear to be available.  Do we do something like that?

Reply:
If it's never genuinely been used, I think it is available?

Reply:
You're right, I'm just projecting problems from C#.